Project Composition (#1584)
bonitasoft/bonita-doc: modules/ROOT/pages/using_Bonita_Studio.adoc

= Using Bonita Studio
Learn how to efficiently use Bonita Studio

== Structure

> By default, here is how the Studio is structured:

Reviewer: ```suggestion
Here is the default Studio structure: 
```

* a top, main menu
* a "coolbar" with generic features

> * a "project explorer" displaying the main elements of the project and opening to the creation of new ones

Reviewer: ```suggestion
* a "project explorer" displaying the main elements of the project, and providing actions to open them or create new ones.
```

* a tabs area with all the files you are currently working on
* a "Problems" area to keep an eye on what needs to be fixed before deployment

> * a dedicated Git section when the Studio is linked to a Git repository

Reviewer: ```suggestion
* a dedicated Git section when the current project is shared with Git.
```

> And then for each tab, some specific areas to interact with. +

Reviewer: ```suggestion
Each tab contains some specific areas to interact with. +
```

You can play with the organization of the tabs and areas. If you get lost, use the "View > Reset view" menu option to come back to default.

== Welcome page

> The Welcome page is displaying upon the first opening of the Studio. +

Reviewer: ```suggestion
The Welcome page is displayed upon the first opening of the Studio. +
```

It gives access to a lot of resources to help you during the lifetime of the project. +

> To come back to it in the middle of the project, click on the "Welcome" icon of the coolbar. 

Reviewer: ```suggestion
The welcome page can be opened at any time by clicking on the "Welcome" icon of the coolbar. 
```

> == Configuration

Reviewer: Do not talk about configuration for preferences, it could lead to misunderstanding with the real configuration concept.

```suggestion
== Preferences
```

> The "Preferences" icon in the studio's coolbar allows a full set of configurations. Find the list and how to use them in xref:bonita-bpm-studio-preferences.adoc[Bonita Studio configuration]

Reviewer: ```suggestion
The "Preferences" icon in the studio's coolbar allows a full set of configurations. Find the list and how to use them in xref:bonita-bpm-studio-preferences.adoc[Bonita Studio preferences]
```
